programs/stake/src/lib.rs

```diff
@@ -14,3 +14,11 @@ pub mod stake_state;
 pub fn add_genesis_accounts(genesis_config: &mut GenesisConfig) -> u64 {
     config::add_genesis_account(genesis_config)
 }
+
+/// The minimum stake amount that can be delegated, in lamports.
+/// NOTE: This is also used to calculate the minimum balance of a stake account, which is the
+/// rent exempt reserve _plus_ the minimum stake delegation.
+#[inline(always)]
+pub fn get_minimum_delegation(_feature_set: &FeatureSet) -> u64 {
```

Comment:
I think this should be at most `pub(crate)`, external consumers will use the stake program instruction

Reply:
Done!